docs/kiali/02-01-kiali.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Architecture
+---
+The following diagram provides an overview of the Kiali architecture in Kyma:
+
+![Kiali architecture](assets/architecture.svg)
+
+1. Kiali is integrated into Kyma's authentication component `dex`, which grants access to the Kiali Dashboards and graphs to authorised users.
```

Comment:
Kiali is not integrated into dex. We have the keycloak-gatekeeper in front of kiali assring that only authenticated requests can reach it. The gatekeeper will verify a presented token on base of the OIDC discovery endpoint of dex, requests with no token will be redirected to dex